syntax = "proto3";

package eu.sotec;

message MqttPublishSkillParams {
    string topic = 1;
    string message = 2;
    bool retained = 3;
}
